src/Microsoft.DotNet.Build.VSTSBuildsApi/VSTSBuildClient.cs

```diff
@@ -0,0 +1,228 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace VSTSBuildsApi
+{
+    public class VSTSBuildClient
```

Comment:
Style guides I've seen say acronyms over two chars should be considered a word, so `VstsBuildClient` and `VstsBuildsApi`. This also lets you rename the private methods from (e.g.) `VSTS_UpdateDefinition` to `VstsUpdateDefinition`, unless there was some other reason for the underscore than readability.